README.md

```diff
@@ -180,3 +180,18 @@ var options = {
 };
 app.use(kayvee.middleware(options));
 ```
+
+You can also log additional logs with the request context using log level functions available on `req.log`:
```

Comment:
nit: maybe

"You can also log with the request context using `req.log`. For example:"